src/stdlib/Localization.ts

```diff
@@ -0,0 +1,89 @@
+import * as fs from "fs";
+import * as path from "path";
+import { promisify } from "util";
+const readFile = promisify(fs.readFile);
+
+import { Interpreter } from "../interpreter";
+import { BrsString, Callable, ValueKind, StdlibArgument, RoDeviceInfo } from "../brsTypes";
+import { XmlDocument } from "xmldoc";
+
+/**
+ * Supported locales in RBI.
```

Comment:
I don't know that we _need_ to enforce this, we could just grab whatever subdirectories exist in the `locale/` directory. But it made traversal a little easier 🤷